skt/executable.py

```diff
@@ -45,7 +45,7 @@ def save_state(config_set, state):
     """
 
     def set_field(key, val, dst):
-        if key in ['type', 'jobtemplate', 'jobowner', 'blacklist']:
+        if key in ['jobtemplate', 'jobowner', 'blacklist']:
```

Comment:
Super nitpick, mostly for the future, not necessarily for this PR: it's more Pythonic to use tuples in this case (throwaway, literal collection) instead of lists.

Reply:
I didn't want to include unrelated changes in the PR :)

Reply:
Whole function dropped by jracek